### PR TITLE
Feature 'aliases documentation'

### PR DESCRIPTION
**Pull request by schalkms**
reference #979

### CODE REVIEW

File: detekt-generator/src/main/kotlin/io/gitlab/arturbosch/detekt/generator/collection/MultiRuleCollector.kt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package io.gitlab.arturbosch.detekt.generator.collection
 
 import io.gitlab.arturbosch.detekt.api.DetektVisitor
+import io.gitlab.arturbosch.detekt.generator.collection.exception.InvalidDocumentationException
```
Comment (vanniktech) [inline detekt-generator/src/main/kotlin/io/gitlab/arturbosch/detekt/generator/collection/MultiRuleCollector.kt]: Seems unrelated

Comment (schalkms) [reply detekt-generator/src/main/kotlin/io/gitlab/arturbosch/detekt/generator/collection/MultiRuleCollector.kt]: Please take a look at the commit 93d9e01.
I moved the documentation generator exceptions to a separate package.

**Comment by arturbosch** [comment]
Good stuff!
